framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -170,6 +170,20 @@ public QualifierDefaults(Elements elements, AnnotatedTypeFactory atypeFactory) {
         this.useUncheckedCodeDefaultsSource = atypeFactory.getContext().getChecker().useUncheckedCodeDefault("source");
     }
 
+    @Override
+    public String toString() {
+        // displays the checked and unchecked code defaults
+        StringBuilder sb = new StringBuilder();
+        sb.append("Checked Code Defaults: ");
+        sb.append(System.lineSeparator());
+        sb.append(PluginUtil.join(System.lineSeparator(), checkedCodeDefaults));
+        sb.append(System.lineSeparator());
+        sb.append("Unchecked Code Defaults: ");
+        sb.append(System.lineSeparator());
+        sb.append(PluginUtil.join(System.lineSeparator(), uncheckedCodeDefaults));
+        return sb.toString();
+    }
```

Comment:
It might be useful to output the boolean useUncheckedCodeDefaultsSource and  useUncheckedCodeDefaultsBytecode.